README.md

```diff
@@ -0,0 +1,16 @@
+:repository-owner: arduino-libraries
+:repository-name: Arduino_PF1550
+
+= {repository-name} Library for Arduino =
+
+image:https://github.com/{repository-owner}/{repository-name}/actions/workflows/check-arduino.yml/badge.svg["Check Arduino status", link="https://github.com/{repository-owner}/{repository-name}/actions/workflows/check-arduino.yml"]
+image:https://github.com/{repository-owner}/{repository-name}/actions/workflows/compile-examples.yml/badge.svg["Compile Examples status", link="https://github.com/{repository-owner}/{repository-name}/actions/workflows/compile-examples.yml"]
+image:https://github.com/{repository-owner}/{repository-name}/actions/workflows/spell-check.yml/badge.svg["Spell Check status", link="https://github.com/{repository-owner}/{repository-name}/actions/workflows/spell-check.yml"]

```

Comment:
You are using [AsciiDoc](https://asciidoc.org/) markup here, but you used the `.md` file extension of the Markdown markup language.

This causes the content to be rendered incorrectly. You can see how it is rendered here:

https://github.com/arduino-libraries/Arduino_PF1550/blob/b8295ddd0cf9d23eb0deec7f75d252f332b55272/README.md

You must use the appropriate markup for the language of the file. You can learn about the Markdown language here:

https://www.markdownguide.org/

Reply:
I assumed that they were a subset of Markdown. I removed the lines entirely.